RFC-0001-Agile Coretime.md

```diff
@@ -0,0 +1,265 @@
+# RFC-1: Agile Coretime
+
+|                 |                                                                                             |
+| --------------- | ------------------------------------------------------------------------------------------- |
+| **Status**      | Draft Proposal                                                                              |
+| **Areas**       | General                                                                                     |
+| **Description** | Agile periodic-sale-based model for assigning Coretime on the Polkadot Ubiquitous Computer. |
+| **Issues**      | n/a                                                                                         | 
+| **Authors**     | Gavin Wood                                                                                  |
+| **Reviewers**   | None                                                                                        |
+
+
+## Summary
+
+This proposes a periodic, sale-based method for assigning and Polkadot Coretime. The method takes into account the need for long-term capital expenditure planning for teams building on Polkadot, yet also provides a means to allow Polkadot to capture long-term value in the resource which it sells. It supports the possibility of building secondary markets to make resource allocation more efficient and largely avoids the need for parameterisation.
+
+## Motivation
+
+### Present System
+
+The present system of allocating time for parachains on the cores of the Polkadot Ubiquitous Computer (aka "Polkadot") is through a process known as *slot auctions*. These are on-chain candle auctions which proceed for several days and result in a core being assigned to a single parachain for 6six months at a time up to 18 months in advance. Practically speaking, we only see two year periods being bid upon and leased.
```

Comment:
Typo: 6six

Reply:
This is definitely going to be HUGE! as the team has been an inspiration in their innovative approaches towards achieving the project goals and visions, I am so glad to be part of this project
This is definitely going to be HUGE! as the team has been an inspiration in their innovative approaches towards achieving the project goals and visions, I am so glad to be part of this project
This is definitely going to be HUGE! as the team has been an inspiration in their innovative ap